lib/core/app/app_router.dart

```diff
@@ -219,19 +229,49 @@ class AppRouter {
           },
         ),
         GoRoute(
-            path: Pages.scanNFC.path,
-            name: Pages.scanNFC.name,
-            builder: (context, state) {
-              return BlocProvider(
-                create: (context) => ScanNfcCubit()
-                  ..startTagRead(delay: ScanNfcCubit.startDelay),
-                child: const NFCScanPage(),
-              );
-            }),
+          path: Pages.scanNFC.path,
+          name: Pages.scanNFC.name,
+          builder: (context, state) {
+            return BlocProvider(
+              create: (context) =>
+                  ScanNfcCubit()..startTagRead(delay: ScanNfcCubit.startDelay),
+              child: const NFCScanPage(),
+            );
+          },
+        ),
         GoRoute(
           path: Pages.successCoin.path,
           name: Pages.successCoin.name,
           builder: (context, state) => const SuccessCoinScreen(),
         ),
+        GoRoute(
+          path: Pages.voucherCodeScreen.path,
+          name: Pages.voucherCodeScreen.name,
+          builder: (context, state) => const VoucherCodeScreen(),
+        ),
+        GoRoute(
+          path: Pages.exhibitionOrganisations.path,
+          name: Pages.exhibitionOrganisations.name,
+          builder: (context, state) {
+            return BlocProvider(
+              create: (context) => OrganisationsCubit(
+                getIt(),
+              )..getRecommendedOrganisations(
+                  location: OrganisationsCubit.exhibitionLocation,
+                  interests: OrganisationsCubit.exhibitionInterests,
+                  // fakeComputingExtraDelay: const Duration(seconds: 1),
```

Comment:
Do you wanna keep this comment or remove it?

Reply:
removed